chrisdhaan/CDYelpFusionKit
# Dependency Updates

### Issue Link :link:
#26 

### Goals :soccer:
Dependency updates

### Implementation Details :construction:
- Update AlamoFire to latest version
- Add `validate` method to AF requests
- Remove `ObjectMapper` in favor of `Decodable` API
- Replace `reponseObject` with `responseDecodable`
- Convert modal classes to structs
- Convert `vars` to `lets` for properties in model structs
- Make sure SwiftPackageManager build works

### Testing Details :mag:
Tested locally
